src/Pages/Home/WelcomeMessage.js

```diff
@@ -0,0 +1,14 @@
+import react from 'react';
```

Comment:
Can you change `import react from 'react';` to `import React from 'react';`. It will prevent a warning from happening every time we run our app.